services/beeper-connector/beeper_to_rdf.py

```
@@ -0,0 +1,183 @@
+
+def get_user_info(cursor, user_id):
+    """Get user information from the database."""
+    try:
+        cursor.execute("SELECT json_extract(user, '$') FROM users WHERE userID = ?", (user_id,))
+        result = cursor.fetchone()
+        if result and result[0]:
+            user_data = json.loads(result[0])
+            name = user_data.get('fullName', user_id)
+            return name
+        return user_id
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)